src/i18n/resources/de/translation.json

```diff
@@ -477,6 +478,77 @@
       "tillering": "Bestockung",
       "tilleringFirwood": "Tannenanteil",
       "tilleringHardwood": "Laubholzanteil",
+      "vegetationIndicator": {
+        "label": "Zeigergruppen",
+        "a": "Starke Säurezeiger, feucht-nass",
+        "b": "Starke Säurezeiger, frisch / trocken",
+        "c": "Schwache Säurezeiger",
+        "e": "Buchenwaldarten",
+        "f": "Nährstoffzeiger, frisch",
+        "g": "Nährstoffzeiger, tonig",
+        "h": "Nährstoffzeiger, feucht",
+        "i": "Nässezeiger",
+        "j": "Hochstauden",
+        "k": "Feuchtezeiger, basenarm",
+        "l": "Montan-Zeiger (Bergstufe)",
+        "m": "Basen- und Kalkzeiger",
+        "n": "Arten auf Blockschutt",
+        "o": "Basenzeiger wechselfeucht / wechseltrocken",
+        "p": "Trockenheitszeiger"
+      },
+      "terrain": "Relief",
+      "cantonalForestType": "Kantonaler Standortstyp"
+    }
+  },
+  "bl": {
```

Comment:
Do we provide translations for BL? For example  the `AssociationsTab` component has untranslated strings. But if BL is only used in German we can skip i18n.

Reply:
@ElkeErhardt Is BL going to be German only?

Reply:
Yes there will be no French Version, same like Luzern

Reply:
I removed all i18n content for Basel and hard-code the text content